src/pages/TypographyExpose.vue

```diff
@@ -0,0 +1,169 @@
+<template>
```

Comment:
keep this page as is, but delete all of the CSS in this page.

Make sure that the global typography css is used instead.